contribs/gmf/src/directives/locationchooser.js

```diff
@@ -24,12 +25,13 @@ gmf.locationchooserDirective = function() {
     restrict: 'E',
     scope: {
       'getMapFn': '&gmfLocationchooserMap',
-      'getLocationsFn': '&gmfLocationchooserLocations'
+      'getLocationsFn': '&gmfLocationchooserLocations',
+      'selectedLocation': '=gmfLocationchooserSelected'
     },
     controller: 'gmfLocationchooserController',
     controllerAs: 'ctrl',
     template: '<select ' +
-        'ng-model="ctrl.selectedLocation" ' +
```

Comment:
You may want to use `bindToController`. In this way you can still use `ctrl.selectedLocation` here.